### DISCUSSION TITLE
Why can't I see animated skin in animated_skin.wbt?

### DISCUSSION THREAD

**Discussion by Kreijstal** [cyberbotics/webots#4742] [Q&A]
![grafik](https://user-images.githubusercontent.com/2415206/175491501-05401ad2-326a-48b6-bbf2-df1e737f1b90.png)
The salamander looks something like this. I compiled this from latest commit on master https://github.com/cyberbotics/webots/commit/eafabe20212300bb19d0a9a70ae5d4918036f9f0

**Comment by omichel** [comment]
Yes, the master branch may be currently broken as we focus our development effort on the develop branch which is going to be used for the next release to be published next week or so. So, I would recommend you to switch to the develop branch where this bug should be fixed.
